Lib/test/test_urllib.py

```diff
@@ -257,6 +261,33 @@ def test_url_fragment(self):
         finally:
             self.unfakehttp()
 
+    @unittest.skipUnless(ssl, "ssl module required")
```

Comment:
is this true for these tests?  (not that it matters, all sane platforms have ssl so these tests will be run regardless)

Reply:
oops, no, the decorator is wrong: i removed it.